app/src/components/inputs/inputRadio/inputRadio.scss

```diff
@@ -57,5 +70,5 @@
   font-size: 13px;
   color: $COLOR--charcoal-grey;
   line-height: 18px;
-  word-break: break-all;
+  word-break: keep-all;
```

Comment:
With localization and wide text in nested span tag message will be look like text in attachment
![image](https://user-images.githubusercontent.com/31861805/41107014-04ec6378-6a7a-11e8-80e1-196728021a07.png)

Reply:
So word-break: keep-all; property fix it and don't smash view in any cases

Reply:
word-break: keep-all property designed for a  Chinese/Japanese/Korean languages. For others it's works like default word-break: normal. In our application, I think, exist both cases: when rediobutton text should be multi-line and shouldn't. So it will be good to have a class-modifier prop in component, which will control word-break property.

Reply:
fixed